test/PspUpf.jl

```diff
@@ -250,3 +250,42 @@ end
         end
     end
 end
+
+@testitem "Test nonlocal term operations" tags=[:psp] setup=[mPspUpf] begin
```

Comment:
Not sure about this test. The goal was just for me to be able to try my changes with <5s waiting time.